### DISCUSSION TITLE
Installing jaxlib with cuda11.cudnn82

### DISCUSSION THREAD

**Discussion by danieldanciu** [jax-ml/jax#11443] [Q&A] [answered]
When I run:
```pip install jaxlib==0.3.7+cuda11.cudnn82 -f https://storage.googleapis.com/jax-releases/jax_releases.html```
I get an error:
```
ERROR: Could not find a version that satisfies the requirement jaxlib==0.3.7+cuda11.cudnn82 (from versions: 0.1.60, 0.1.61, 0.1.62, 0.1.63, 0.1.64, 0.1.65, 0.1.66, 0.1.67, 0.1.68, 0.1.69, 0.1.70, 0.1.71, 0.1.72, 0.1.73, 0.1.74, 0.1.75, 0.1.76, 0.3.0, 0.3.2, 0.3.5, 0.3.7, 0.3.8, 0.3.10, 0.3.14)
```
I am running on a Linux machine with python 3.7 and pip 22.1.2. I am quite sure this used to work until recently and I can't think of any changes in my environment. Installing jaxlib without the cuda extensions works just fine. Similarly, if I directly provide a wheel, I can also install with no issues:
```pip install https://storage.googleapis.com/jax-releases/cuda11/jaxlib-0.3.5+cuda11.cudnn82-cp37-none-manylinux2010_x86_64.whl```

But there are no cuda11 wheels for any version > 0.3.5 - again I am quite sure there used to be. 
Strangely, I can find no cuda wheels here either: https://storage.googleapis.com/jax-releases/jax_releases.html
What am I missing?

**Comment by hawkinsp** [comment, marked as answer]
The path to the `cuda` wheels changed, see: https://github.com/google/jax#pip-installation-gpu-cuda

(Sorry, splitting the CPU and GPU wheels has broken things because the path has changed. But we're not going to change it back now because that would be more disruptive.)

**Comment by danieldanciu** [comment]
That was it, thank you! No amount of Googling led me to that README page...
